src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -275,14 +274,6 @@ public Path getFulltextIndexPath() {
         return indexPath;
     }
 
-    public String getUniqueName() {
-        if (getDatabasePath().isPresent()) {
-            Path databasePath = getDatabasePath().get();
-            return BackupFileUtil.getUniqueFilePrefix(databasePath) + "--" + databasePath.getFileName();
-        }
-        return "unsaved" + NUMBER_OF_UNSAVED_LIBRARIES++;
-    }
-
```

Comment:
That this is removed explains a lot - I missed "my" numbers at untitled - but it is OK IMHO.

The other unique path feature still works:

![image](https://github.com/user-attachments/assets/18a6749c-36c3-4e86-8851-45a8287c2ca4)

Refs https://github.com/JabRef/jabref-issue-melting-pot/issues/614

Reply:
No, this method was used only for the Postgres table name, so it is another issue.